src/main/java/com/neuvector/Scanner.java

```diff
@@ -258,10 +237,27 @@ private static String runCMD(String[] cmdArgs, Logger log){
             // Read the output from the command
             String s = null;
             StringBuilder sb = new StringBuilder(String.join(" ", cmdArgs));
+            boolean catScanResultJsonOutput = false;
             while ((s = stdInput.readLine()) != null) {
+                if (outputCollector != null) {
+                    outputCollector.append(s);
+                }
                 // sb.append(System.getProperty("line.separator"));
                 if (log != null) {
-                    log.info(s);
+                    if ("scan_result.json".equals(s)) {
```

Comment:
Comments should make it clear what is happening here. 

When `cat scan_result.json` is called, we stop logging unless debug is enabled. Otherwise we would log `scan_result.json` contents too. That json can be quite large. 

To also log `scan_result.json` log level needs to be set to debug. This can be done by passing `-X` to nexus-iq-cli. 

This is what you see in the logs in nexus-iq-cli when log level is **not** debug:
```
[INFO] Image: https://registry.hub.docker.com/library/alpine:latest
[INFO] Base OS: alpine:3.20.3
[INFO] Created at: 2024-09-06T22:20:07Z
[INFO] 
[INFO] Vulnerabilities: 0, HIGH: 0, MEDIUM: 0, LOW: 0, UNKNOWN: 0
[INFO] If debug mode is enabled, the contents of scan_result.json can be observed in the logs.
[INFO] Contents of scan_result.json are not included at info level.
```

This is what you see when it **is** debug:
```
[INFO] Vulnerabilities: 0, HIGH: 0, MEDIUM: 0, LOW: 0, UNKNOWN: 0
[DEBUG] scan_result.json
[DEBUG] {
[DEBUG]     "error_message": "",
[DEBUG]     "report": {
[DEBUG]         "image_id": "91ef0af61f39ece4d6710e465df5ed6ca12112358344fd51ae6a3b886634148b",
[DEBUG]         "registry": "https://registry.hub.docker.com",
[DEBUG]         "repository": "library/alpine",
[DEBUG]         "tag": "latest",
[DEBUG]         "digest": "sha256:33735bd63cf84d7e388d9f6d297d348c523c044410f553bd878c6d7829612735",
[DEBUG]         "size": 7792915,
[DEBUG]         "author": "",
[DEBUG]         "base_os": "alpine:3.20.3",
[DEBUG]         "created_at": "2024-09-06T22:20:07Z",
[DEBUG]         "cvedb_version": "3.528",
[DEBUG]         "cvedb_create_time": "2024-08-12T00:27:20Z",
[DEBUG]         "layers": [],
```

I am not printing the whole output, I think it is clear what I am trying to explain.
```